Gemfile.lock

```diff
@@ -2,33 +2,30 @@ PATH
   remote: .
   specs:
     xcflushd (1.0.0)
-      3scale_client (~> 2.10)
-      concurrent-ruby (= 1.0.2)
-      daemons (= 1.2.4)
-      gli (= 2.14.0)
-      hiredis (= 0.6.1)
-      net-http-persistent (= 2.9.4)
-      redis (= 3.3.2)
+      3scale_client (~> 2)
+      concurrent-ruby (~> 1)
+      daemons (~> 1.2)
+      gli (~> 2.15)
+      net-http-persistent (~> 2)
+      redis (~> 3.3)
 
 GEM
   remote: https://rubygems.org/
   specs:
-    3scale_client (2.10.0)
+    3scale_client (2.11.0)
       nokogiri
     ast (2.3.0)
-    concurrent-ruby (1.0.2)
+    concurrent-ruby (1.0.5-java)
     daemons (1.2.4)
     diff-lcs (1.2.5)
     docile (1.1.5)
     fakeredis (0.6.0)
       redis (~> 3.2)
-    gli (2.14.0)
-    hiredis (0.6.1)
+    gli (2.16.0)
     json (2.0.2)
-    mini_portile2 (2.1.0)
+    json (2.0.2-java)
```

Comment:
There's json (2.0.2) and json (2.0.2-java). This could be a problem. Running 'bundle install' inside the development container generates a Gemfile.lock with only the -java version.